kubernetes/job-db-migrate-staging.tmpl

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: panoptes-migrate-db-staging
+spec:
+  template:
+    spec:
+      containers:
+      - name: panoptes-migrate-db-staging
+        image: zooniverse/panoptes:__IMAGE_TAG__
+        command: ["/rails_app/migrate.sh"]
+        envFrom:
+        - secretRef:
+            name: panoptes-common-env-vars
+        - secretRef:
+            name: panoptes-staging-env-vars
+        - configMapRef:
+            name: panoptes-staging-shared
+      restartPolicy: Never
+  backoffLimit: 4
```

Comment:
I was assuming this would be 1. Are we retrying 4 times in case of transient failures? if so should we stop after 2?